### PR TITLE
Changes to the Fist Spell

### PR DESCRIPTION
**Pull request by Arthurdentist**
The Fist Spell (alongside Push) is probably one of the least interesting spells to be up against. Its problems come from the fact that it is extraordinarily cheap, but it provides you with very easy stuns. It isn't a spell you can "run from" easily, because the wizard doesn't need to be near you to cast it, and combined with the scrying orb it can be cast through walls. Furthermore it can be bought for the same prize as the Shoe Snatch charm and other weak spells, and that means it isn't too expensive to upgrade all the way. Combined with the mutate spell, it becomes a nightmare to do anything against.

Two Changes I have made are as follows:
1. Fist now costs the same as a standard spell - I think this is the most important of the two changes. This makes it much more costly to upgrade, and wizards will have to sacrifice something else if they want to get Fist to its maximum potential.

2. Fist now targets three people at a time - I am not particularly married to this change, but it does prevent you from clearing entire rooms of people without facing any real consequences. You would no longer be able to beat on everybody in medbay or security, meaning you might have to think a bit more about where you spam the spell, and it would give other people in crowded rooms an opportunity to react. Three targets is still a lot of targets. 

Obviously balance changes are always controversial, and if you think I'm a fucking idiot or you simply disagree with the direction I want to take this spell, please post below. These are just some of the things that I **personally** believe would be better, but your millage may vary.  

TL;DR Admin wants to nerf spell because he hates fun

### CODE REVIEW

**Comment by Arthurdentist** [comment]
here is some previous discussion on the topic #16563

**Comment by ShiftyRail** [comment]
The absolute madman

**Review by PJB3005 — APPROVED** [review]

**Comment by Cyadox** [comment]
If people are upset about the amount of targets, you could have it ramp up with the number of upgrades, or have it inversely proportional to damage/stun time - i.e. less upgrades = less damage = more targets, more upgrades = more damage = less targets.

Regardless, while I haven't been on the receiving or giving end, this seems reasonably more balanced than the current fist spell.

**Comment by AutismSpider** [comment]
Give fist an upgrade tree 
Power = More damage 
Dexterity  = can hit more targets
Agility = can get multihits on targets or lowers the spells cool down

**Comment by Killette2** [comment]
haha now I can carry a locker full of monkeys to counter fist!
